sktime/forecasting/base/_base.py

```diff
@@ -1251,42 +1275,78 @@ def _predict_interval(self, fh, X=None, coverage=0.90):
            Forecasting horizon, default = y.index (in-sample forecast)
         X : pd.DataFrame, optional (default=None)
            Exogenous time series
-        alpha : float or list, optional (default=0.95)
-           Probability mass covered by interval or list of coverages.
+        coverage : float or list, optional (default=0.95)
+           nominal coverage(s) of predictive interval(s)
 
         Returns
         -------
         pred_int : pd.DataFrame
             Column has multi-index: first level is variable name from y in fit,
-                second level being quantile fractions for interval low-high.
-                Quantile fractions are 0.5 - c/2, 0.5 + c/2 for c in coverage.
-            Row index is fh. Entries are quantile forecasts, for var in col index,
-                at quantile probability in second col index, for the row index.
+                second level coverage fractions for which intervals were computed.
+                    in the same order as in input `coverage`.
+                Third level is string "lower" or "upper", for lower/upper interval end.
+            Row index is fh. Entries are forecasts of lower/upper interval end,
+                for var in col index, at nominal coverage in second col index,
+                lower/upper depending on third col index, for the row index.
+                Upper/lower interval end forecasts are equivalent to
+                quantile forecasts at alpha = 0.5 - c/2, 0.5 + c/2 for c in coverage.
         """
-        alphas = []
-        for c in coverage:
-            alphas.extend([(1 - c) / 2.0, 0.5 + (c / 2.0)])
-        alphas = sorted(alphas)
-        pred_int = self._predict_quantiles(fh=fh, X=X, alpha=alphas)
-        pred_int = pred_int.rename(columns={"Quantiles": "Intervals"})
+        implements_interval = self._has_implementation_of("_predict_interval")
+        implements_quantiles = self._has_implementation_of("_predict_quantiles")
+        implements_proba = self._has_implementation_of("_predict_proba")
+        can_do_proba = implements_interval or implements_quantiles or implements_proba
+
+        if not can_do_proba:
+            raise RuntimeError(
+                f"{self.__class__.__name__} does not implement "
+                "probabilistic forecasting, "
+                'but "capability:predict_int" flag has been set to True incorrectly. '
+                "This is likely a bug, please report, and/or set the flag to False."
+            )
```

Comment:
should this not go into a unit test instead here?

Reply:
yes, good idea

Reply:
added in https://github.com/alan-turing-institute/sktime/pull/1961